platform/linuxbsd/x11/display_server_x11.h

```diff
@@ -64,6 +64,8 @@
 #include "../freedesktop_screensaver.h"
 #endif
 
+// X11's Time type conflicts with Godot's Time class, so we need to rename it.
+#define Time TimeX11
```

Comment:
Not really happy about that.

I'm even not sure how it works - you're overriding `Time` with a type which doesn't exist.

Reply:
It defines Time as TimeX11, so `typedef unsigned long Time;` in the X11 header becomes `typedef unsigned long TimeX11;`

Without this, we would have to keep a copy of the enums defined in OS since we can't include the Time class.

Also, this approach is already used in the files in the dynwrappers folder.